tools/check_ts_files.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -e
+
+TEMP_FILES=()
+
+cleanup() {
+  for file in "${TEMP_FILES[@]}"; do
+    if [[ -f "$file" ]]; then
+      rm "$file"
+    fi
+  done
+}
+
+trap cleanup EXIT
+
+function check_translation_file() {
+  if [ $# -ne 1 ]; then
+    echo "Usage: $0 <translation_file.ts>"
+    exit 1
+  fi
+  local ts_file=$1
+
+  echo "Checking '$ts_file' is up to date..."
+  cp "$ts_file" "${ts_file/.ts/.a.ts}"
+  TEMP_FILES+=("${ts_file/.ts/.a.ts}")
+  cp "$ts_file" "${ts_file/.ts/.b.ts}"
+  TEMP_FILES+=("${ts_file/.ts/.b.ts}")
+  #
+  pylupdate5 "$LABELME_DIR/*.py" "$LABELME_DIR/widgets/*.py" -ts "${ts_file/.ts/.b.ts}"
+  diff <(sed -r 's/line="[0-9]+"//g' "${ts_file/.ts/.a.ts}") <(sed -r 's/line="[0-9]+"//g' "${ts_file/.ts/.b.ts}")
```

Comment:
I'm not sure I'm following what's happening here. Why is the pylupdate5 line twice here, aside from quoting?

Reply:
is it twice? I am intended to call only once:

<img width="1730" height="220" alt="image" src="https://github.com/user-attachments/assets/5ff83aa2-929b-44c7-9ce8-68ff54c129e2" />